dev/com.ibm.ws.jaxrs.2.1.sse_fat/publish/servers/com.ibm.ws.jaxrs21.sse.jaxb/server.xml

```diff
@@ -13,6 +13,8 @@
 		<feature>componenttest-1.0</feature>
 		<feature>jaxrs-2.1</feature>
 		<feature>osgiConsole-1.0</feature>
+        <feature>xmlBinding-3.0</feature>
+        <feature>timedexit-1.0</feature>
```

Comment:
You shouldn't need the timedexit feature - I think componenttest-1.0 includes it automatically.